0006-cmx2cdp-protocol/README.md

```diff
@@ -1,84 +1,92 @@
 ---
-name: CMX to CDP protocol
-shortname: 6/CMX2CDP
-status: raw
+name: Configuration Data Transport protocol
+shortname: 6/CDT
```

Comment:
I'm afraid that a new slew of 3-letter acronyms will add confusion to the already confusing world of services. Can we improve somehow?